src/Hyperion/Main.hs

```diff
@@ -70,6 +71,7 @@ data Config = Config
   , configRaw :: Bool
   , configSamplingStrategy :: SamplingStrategy
   , configUserMetadata :: JSON.Object
+  , configSelector :: Maybe (BenchmarkId -> Bool)
```

Comment:
This type looks redundant. Why have a `Maybe` when you can craft a function that constantly returns `True`?